src/devices/bus/ti99/gromport/cartridges.cpp

```diff
@@ -239,7 +239,7 @@ image_init_result ti99_cartridge_device::call_load()
 		}
 		catch (rpk_exception& err)
 		{
-			logerror("Failed to load cartridge '%s': %s\n", basename(), err.to_string());
```

Comment:
`logerror` and friends can handle std::string fine - please don't go adding `.c_str()` unnecessarily, it actually hurts performance.